src/main/java/com/codeborne/selenide/SelenideTargetLocator.java

```diff
@@ -134,13 +135,19 @@ public Alert alert() {
   }
 
   /**
-   * Switch to the inner frame (last child frame in given sequence)
+   * Switch to the inner frame (last child frame in given sequence).
+   * <p>
+   * This method
+   * <ol>
+   *  <li> first switches to the root frame (aka "default content"),</li>
```

Comment:
- For the ordered list it's unnecessary to use "first...", "then..."
- `firstFame` -> `firstFrame`, `otherFrames` -> `nextFrames`
- JavaDoc doesn't recognize backticks, it's better to use `<code>`, `{@code}`, or simply regular `""`
- `3. consecutively switches to "nextFrames" until it reaches the last one`

Reply:
thanks, fixed.